snaps/qemu-op-tee-gadget/cloud.conf

```diff
@@ -0,0 +1,11 @@
+datasource_list: [ None ]
+datasource:
+  None:
+    userdata_raw: |
+      #cloud-config
+      users:
+      - name: ubuntu
+        sudo: "ALL=(ALL) NOPASSWD:ALL"
+        lock_passwd: false
+        shell: /bin/bash
+        passwd: "$6$rounds=4096$/VbyQIx0iU635Ak6$0SKYZPoTzACvKN5TdLm5BuUrnOuv6QjHQXEi9d.pGqZVdnUkR3KqqVnXKGj6vlz0LpfOQhY17saDzxul9Zvil."
```

Comment:
cloud-init has alsoe a way to set a plain test password:
```
plain_text_passwd: 'ubuntu'
```
probably better for readability, although tbh I have not tried it yet.